src/platform/win32localfile.cpp

```diff
@@ -189,18 +189,14 @@ bool Win32LocalFile::Scan_Int(int &integer)
     while (true) {
         number.Concat(tmp);
 
-        int bytes = read(m_fileHandle, &tmp, sizeof(tmp));
-
-        if (bytes == 0) {
+        if (read(m_fileHandle, &tmp, sizeof(tmp)) == 0) {
             break;
         }
 
         if (!isdigit(tmp)) {
             // If we read a byte, seek back that byte for the next read
             // as we are done with the current number.
-            if (bytes) {
```

Comment:
I removed this `bytes` test here, because just a few lines earlier it already tests `bytes`. And then simplified the above condition to less lines accordingly, because the `bytes` variable is no longer needed. The same applies to 2 more similar functions below.